TESTING.md

```diff
@@ -14,11 +14,55 @@ In this article you will:
   - Generate C# projects corresponding to all the templates in various configurations (no auth, single-org, single-org calling graph, single-org calling web API, Individual B2C, Individual B2C calling web API (for the web API and the Blazorwasm hosted templates, as B2C does not support OBO)).
   - Configure the projects with existing Azure AD and B2C apps and client secrets. This is done by a configuration file named `configuration.json`. You will need to add the client secrets (see below).
   - Build the generated projects (which are grouped in a solution named `test.sln`).
-- Manually test (for now) the generated projects.
+- [Manually test (for now) the generated projects](How-to-test-the-configured-projects-manually).
 
-## How to generate the test projects
+## How to generate the test projects from a NuGet Package
+> For example, testing for a release with a build from AzureDevOps. 
 
-> If testing a release build, start with step 6 in "Testing the templates from a Nuget package", then go to step 4, and add the secrets.
+In a Developer Command Prompt:
+
+1. Do a git clone of the repostitory into a short file path.
+
+    `cd C:\`
+
+    `mdkir git`
+
+    `cd C:\git`
+
+    `git clone https://github.com/AzureAD/microsoft-identity-web idweb`
+
+2. cd to the root of the repo (for instance `cd C:\git\idweb`)
+
+3. Set the version of the templates to test.
+
+   `Set ClientSemVer=1.1.0`
+
+4. Add client secrets (or your own config file) to the Configuration.json file
+
+   `"B2C_Client_ClientSecret": "sercret_goes_here",`
+
+   `"AAD_Client_ClientSecret": "sercret_goes_here",`
+  
+   `"AAD_WebApi_ClientSecret": "sercret_goes_here"`
+
+5. Build the repo. Copy the NuGet package containing the templates (Microsoft.Identity.Web.ProjectTemplates.version.nupkg) downloaded from the release build and paste it under the `ProjectTemplates\bin\Debug` folder of the repo.
```

Comment:
Do we need to build the repo?

Reply:
yes because otherwise there won't be a bin\debug folder